### PR TITLE
Msw 103814952 update incentive permissions

### PR DESCRIPTION
**Pull request by michaelwehrley**

### CODE REVIEW

**Comment by michaelwehrley** [comment]
@ericcf see your original comments and my responses at https://github.com/cbitstech/sunnyside/pull/172

File: app/controllers/behaviors_controller.rb [hunk 1]
```diff
@@ -66,19 +70,28 @@ def delete_restriction_message
   end
 
   def set_behavior
-    @behavior = Sunnyside::Behavior.find(params[:id])
+    @behavior ||= @incentive.behaviors.find(params[:id])
+    rescue
```
Comment (ericcf) [inline app/controllers/behaviors_controller.rb]: Should this `rescue` from a specific exception(s)?

**Comment by ericcf** [comment]
nice! :shaved_ice:

File: app/controllers/behaviors_controller.rb (hunk 1 above)
Comment (michaelwehrley) [inline app/controllers/behaviors_controller.rb]: @ericcf Thank you: https://github.com/cbitstech/sunnyside/pull/181/files#diff-69f0edd64c2ebbda9c57d457c8fe2b04R74

**Pull request merged**